generate_solve_rllib_test.py

```diff
@@ -536,6 +536,10 @@ def _get_action_space_(self) -> GymSpace[BelugaGymCompatibleDomain.T_event]:
     classic = not args.numeric
     max_simulation_steps = args.max_simulation_steps
 
+    if args.probabilistic and args.probabilistic_model == "ppddl" and classic:
```

Comment:
Shouldn't it be `and not classic`? @arashHaratian

Reply:
Yes! Sorry it is my mistake. fixed it now X)